packages/components/src/components/button/button.jsx

```diff
@@ -67,13 +83,18 @@ Button.propTypes = {
     children         : PropTypes.node,
     className        : PropTypes.string,
     classNameSpan    : PropTypes.string,
+    green            : PropTypes.bool,
     has_effect       : PropTypes.bool,
     icon             : PropTypes.node,
     id               : PropTypes.string,
     is_disabled      : PropTypes.oneOfType([PropTypes.number, PropTypes.bool]),
     is_loading       : PropTypes.oneOfType([PropTypes.object, PropTypes.bool]),
     is_submit_success: PropTypes.bool,
     onClick          : PropTypes.func,
+    primary          : PropTypes.bool,
+    secondary        : PropTypes.bool,
+    size             : PropTypes.string,
```

Comment:
```suggestion
    size             : PropTypes.oneOf(['s', 'm', 'l']),
```

Reply:
we will change the size to `small`, 'medium', or 'large'